### PR TITLE
Do not wildcard queries for numeric fields of Qtl model

### LINKED ISSUE #253: Do not wildcard numeric fields in global search
Example: `map_locus.map_locus = "102.5"`

This should come up in search by "102.5" but it should not when one looks for "2.5" or "2". I can supply you with a set of fields that should be treated this way.

Wait for #242 before doing that. Also related to #225.

### PR DESCRIPTION
**Pull request by kammerer**
Fixes #253. Should be pretty easy to implement for other models. ES index needs to be rebuilt to check how it works.

### CODE REVIEW

**Comment by Nuanda** [comment]
Tested, works for me. I will add further fields to treat them this way in the issue discussion.

**Comment by kammerer** [comment]
Updated models searchable by `map_positions.map_position`.

File: app/models/concerns/searchable.rb [hunk 1]
```diff
@@ -30,4 +34,10 @@ def as_indexed_json(options = {})
       as_json(self.class.indexed_json_structure)
     end
   end
+
+  def self.classes
+    ActiveRecord::Base.descendants.select { |klass|
+      klass.ancestors.include?(Searchable)
+    }
```
Comment (Nuanda) [inline app/models/concerns/searchable.rb]: Let's use do/end for multiline blocks, and reserve multiline { } use only for hash structures. Ok?

Comment (kammerer) [inline app/models/concerns/searchable.rb]: Fixed

**Comment by Nuanda** [comment]
:+1:

**Pull request merged**